apps/teams-test-app/src/App.tsx

```diff
@@ -129,70 +131,80 @@ export const generateRegistrationMsg = (changeCause: string): string => {
-        <PagesConfigAPIs />
-        <PagesCurrentAppAPIs />
-        <PagesTabsAPIs />
-        <PeopleAPIs />
-        <PrivateAPIs />
-        <ProfileAPIs />
-        <RemoteCameraAPIs />
-        <SearchAPIs />
-        <SecondaryBrowserAPIs />
-        <SharingAPIs />
-        <WebStorageAPIs />
-        <StageViewAPIs />
-        <TeamsCoreAPIs />
-        <TeamsAPIs />
-        <ThirdPartyCloudStorageAPIs />
-        <VideoAPIs />
-        <VideoExAPIs />
-        <VisualMediaAPIs />
-      </div>
-      <Version />
-    </div>
+    <BrowserRouter>
+      <Routes>
+        <Route
+          path="/"
+          element={
+            <>
+              <div className="App-container">
+                <AppAPIs />
+                <AppInitializationAPIs />
+                <AppInstallDialogAPIs />
+                <AuthenticationAPIs />
+                <AppEntityAPIs />
+                <BarCodeAPIs />
+                <CalendarAPIs />
+                <CallAPIs />
+                <ChatAPIs />
+                <ClipboardAPIs />
+                <CookieAccessComponent />
+                <CustomAPIs />
+                <DialogAPIs />
+                <DialogCardAPIs />
+                <DialogCardBotAPIs />
+                <DialogUpdateAPIs />
+                <DialogUrlAPIs />
+                <DialogUrlBotAPIs />
+                <ExternalAppAuthenticationAPIs />
+                <ExternalAppCardActionsAPIs />
+                <ExternalAppCommandsAPIs />
+                <FilesAPIs />
+                <FullTrustAPIs />
+                <GeoLocationAPIs />
+                <Links />
+                <LocationAPIs />
+                <LogAPIs />
+                <MailAPIs />
+                <MarketplaceAPIs />
+                <MediaAPIs />
+                <MeetingAPIs />
+                <MeetingRoomAPIs />
+                <MenusAPIs />
+                <MessageChannelAPIs />
+                <MonetizationAPIs />
+                <NestedAppAuthAPIs />
+                <NotificationAPIs />
+                <OtherAppStateChangedAPIs />
+                <PagesAPIs />
+                <PagesAppButtonAPIs />
+                <PagesBackStackAPIs />
+                <PagesConfigAPIs />
+                <PagesCurrentAppAPIs />
+                <PagesTabsAPIs />
+                <PeopleAPIs />
+                <PrivateAPIs />
+                <ProfileAPIs />
+                <RemoteCameraAPIs />
+                <SearchAPIs />
+                <SecondaryBrowserAPIs />
+                <SharingAPIs />
+                <WebStorageAPIs />
+                <StageViewAPIs />
+                <TeamsCoreAPIs />
+                <TeamsAPIs />
+                <ThirdPartyCloudStorageAPIs />
+                <VideoAPIs />
+                <VideoExAPIs />
+                <VisualMediaAPIs />
+              </div>
+              <Version />
+            </>
+          }
+        />
+        <Route path="/second-page" element={<EmptyPage />} />
```

Comment:
What's your thoughts on renaming this route path to be `/empty` or `/empty-page` to be more descriptive. 

I don't mind keeping it as `second-page` just wanted to get your opinion on this

Reply:
Yeah i was thinking about the naming, maybe empty is right either because i'm including the appApi's in it since i want to use the getcontext call in e2e testing

Reply:
I'm going to go with empty-page here